lib/basedriver/commands/images.js

```diff
@@ -45,14 +46,22 @@ commands.compareImages = async function compareImages (mode, firstImage, secondI
     case MATCH_TEMPLATE_MODE.toLowerCase():
       // firstImage/img1 is the full image and secondImage/img2 is the partial one
       result = await imageUtil.getImageOccurrence(img1, img2, options);
+
+      if (options.multiple) {
+        for (const element of result.multiple) {
+          if (!_.isEmpty(element.visualization)) {
+            element.visualization = element.visualization.toString('base64');
+          }
+        }
+
+        return result.multiple;
+      }
       break;
     default:
       throw new errors.InvalidArgumentError(`'${mode}' images comparison mode is unknown. ` +
         `Only ${JSON.stringify([MATCH_FEATURES_MODE, GET_SIMILARITY_MODE, MATCH_TEMPLATE_MODE])} modes are supported.`);
   }
-  if (!_.isEmpty(result.visualization)) {
```

Comment:
are you sure this should be removed?

Reply:
you're correct, this is a mistake

Reply:
the block is still removed. Please keep it and move the base64 conversion from the `MATCH_TEMPLATE_MODE` case block here if a single element is found

Reply:
think this is fixed now?

Reply:
yes, thanks